### ISSUE TITLE
Get prerelease versions from github

### ISSUE THREAD

**Issue by razzeee** [fedora-infra/anitya#1499]
Github supports tagging releases with a prerelease flag. It would be helpful to fetch prerelease info from there instead of having to setup a filter.

One example: github.com/flatpak/flatpak/releases

**Comment by romulasry** [comment]
It does by default.

**Comment by razzeee** [comment]
Then something about the config of https://release-monitoring.org/project/6377/ is wrong, as it doesn't do that. Can you help with that?

**Comment by romulasry** [comment]
I don't see any of them taged as prerelease.

**Comment by razzeee** [comment]
Yeah, that's the problem, according to github, they should be https://github.com/flatpak/flatpak/releases

![Screenshot_20221030-003043_Firefox Nightly.png](https://user-images.githubusercontent.com/5943908/198854811-b425fdb3-3fcf-4b5f-b8a5-1315b6984704.png)

**Comment by Zlopez** [comment]
If this information is obtainable through Github GraphQL API we can definitely use it, otherwise I don't think we can obtain it.

**Comment by romulasry** [comment]
It seemed working in another one. Can someone force refresh this?

**Comment by razzeee** [comment]
Seems okay now
